code/_onclick/item_attack.dm

```diff
@@ -82,7 +82,7 @@
 	log_combat(user, M, "attacked", src.name, "(INTENT: [uppertext(user.a_intent)]) (DAMTYPE: [uppertext(damtype)])")
 	add_fingerprint(user)
 
-	user.adjustStaminaLossBuffered(getweight())//CIT CHANGE - makes attacking things cause stamina loss
+	user.adjustStaminaLossBuffered(getweight()*0.8)//CIT CHANGE - makes attacking things cause stamina loss

```

Comment:
You should make it a define and add it to `code\__DEFINES\combat.dm`.

Reply:
I'm going off of what was used in the lower part of the code, where the stamloss to attack an object is *1.2 of the weapon weight.

Reply:
Well then, scratch my previous suggestion, the whole item attack "total mass" stamina cost feature is so uncharming and I don't get the point of why attack_obj should be more tiresome.

What I wanted was fewer magic numbers and more coherence. But the feature is slim and rudimentary, and it's not worth a drama.

Reply:
Yeah, it was either, make both of them variables that are used in one spot, or just accept that they're a modifier and go on from there.